src/Private/Helpers.ps1

```diff
@@ -3748,4 +3748,21 @@ function Resolve-PodeObjectArray {
         # For any other type, convert it to a PowerShell object
         return [pscustomobject]$Property
     }
+}
+
+
+function Get-PodeOrderedDictionaryClone {
```

Comment:
Better name might be `Copy-PodeOrderedDictionary`

Reply:
I forgot to replace this function with a more advanced that I'm using in AsyncRoute.
Copy-PodeDeepClone is more powerfull and not limited to dictionary only